modules/ROOT/pages/to-request-vpc-connectivity.adoc

```diff
@@ -3,39 +3,77 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-How you request Anypoint Virtual Private Cloud (Anypoint VPC) connectivity to your network varies depending on the connectivity method. 
+The process for requesting or updating Anypoint Virtual Private Cloud (Anypoint VPC) connectivity to your network depends on the connectivity method you use: IPsec, VPC peering, or Direct Connect.
 
-[%header]
-|===
-|VPC Connection Method|How to
-|IPsec tunnel
-a|
+[NOTE]
+The SLA for provisioning an Anypoint VPC connection is five business days after MuleSoft Support receives and approves your completed VPC discovery form.
 
-* For VPNs created after November 2018, follow the self-service steps in xref:vpn-create-arm.adoc[Create an Anypoint VPN]. +
-* For VPNs created before November 2018, follow the steps in <<request_vpc_connectivity,Request Connectivity for Anypoint VPCs>>.
-|VPC peering
-|<<request_vpc_connectivity,Request Connectivity for Anypoint VPCs>>
-|CloudHub direct connect
-|<<request_vpc_connectivity,Request Connectivity for Anypoint VPCs>>
-|===
 
-[[request_vpc_connectivity]]
-== Request Connectivity for Anypoint VPCs
+== Update Connectivity for Existing IPsec VPN Connections
 
-To connect your Anypoint VPC to your internal network using VPN tunnel, peering, or direct connect, you must:
+As of November 2018, MuleSoft Support no longer creates new VPN connections.
+For all new VPNs, you must follow the self-service steps in xref:vpn-create-arm.adoc[Create an Anypoint VPN].
 
-. Download the link:https://docs.mulesoft.com/downloads/runtime-manager/VPC-form-v9.4-template.xlsx[Anypoint VPC discovery form], which requires at least MS Excel 2007 (with macros enabled). 
+To update *existing* IPsec VPN connections that were created by MuleSoft Support before November 2018:
+
+. Download the link:https://docs.mulesoft.com/downloads/runtime-manager/VPC-form-v9.4-template.xlsx[Anypoint VPC discovery form].
```

Comment:
This is the line that started all the trouble: Is the form for discovering ANY VPCs or only Anypoint VPCs?
Please search for "VPC" and evaluate whether your intention is clear. I tried, but I was not sure enough to commit....